lib/core/src/send_swap.rs

```diff
@@ -93,14 +94,29 @@ impl SendSwapHandler {
                 Ok(())
             }
 
-            // Boltz has detected the lockup in the mempool, we can speed up
-            // the claim by doing so cooperatively
+            // Boltz has detected the lockup in the mempool. If the swap is not to be batched
+            // we can speed up the claim by doing so cooperatively.
             SubSwapStates::TransactionClaimPending => {
                 if swap.metadata.is_local {
-                    self.cooperate_claim(&swap).await.map_err(|e| {
-                        error!("Could not cooperate Send Swap {id} claim: {e}");
-                        anyhow!("Could not post claim details. Err: {e:?}")
-                    })?;
+                    let claim_tx_response = self.swapper.get_send_claim_tx_details(&swap).await?;
```

Comment:
Just verifying this will not return error in case of batched claim (since the swapper won't want to cooperate on claim here).

Reply:
It was returning an error, fixed [4208d70](https://github.com/breez/breez-sdk-liquid/pull/863/commits/4208d701b7f527440b438c8a8259259a6531550f)